assets/js/modules/reader-revenue-manager/utils/validation.test.js

```diff
@@ -33,15 +33,32 @@ describe( 'utility functions', () => {
 			[ 'false', false ],
 			[ 'an integer', 12345 ],
 			[ 'an empty string', '' ],
-			[ 'a string with invalid characters', 'invalid-publication!ID' ],
+			[ 'a string with invalid characters', 'invalid.publication!ID' ],
+			[ 'a string with periods', 'invalid.publication.ID' ],
 		] )( 'should return FALSE when %s is passed', ( _, publicationID ) => {
 			expect( isValidPublicationID( publicationID ) ).toBe( false );
 		} );
 	} );
 
 	describe( 'isURLUsingHTTPS', () => {
+		beforeAll( () => {
+			jest.spyOn( console, 'warn' ).mockImplementation( () => {} );
+		} );
+
+		afterAll( () => {
+			// eslint-disable-next-line no-console
+			console.warn.mockRestore();
+		} );
+
+		afterEach( () => {
+			// eslint-disable-next-line no-console
+			console.warn.mockClear();
+		} );
+
```

Comment:
`console.warn()` is already spied on by `@wordpress/jest-console`, this explicit spying isn't needed.

```suggestion
```

Reply:
Updated.